BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeHttpClient.kt

```diff
@@ -91,19 +91,19 @@ internal class BraintreeHttpClient(
         configuration: Configuration?,
         authorization: Authorization?,
         additionalHeaders: Map<String, String> = emptyMap(),
-        callback: HttpResponseCallback
+        callback: NetworkResponseCallback?
```

Comment:
Does this need to be a nullable callback now? This leaves the possibility that we will just return with no callback at all from this post request

Reply:
I actually suggested this one. We originally had `HttpNoResponseCallback` that would no-op, but a nullable callback conveys the same meaning. It doesn't have a default arg so an explicit `null` will need to be passed. Maybe there's a better way does it make sense to revert to `HttpNoResponseCallback`?

Reply:
Ahh ok I think I was missing that this is callback is only null for analytics requests where it would be ok if we had a no-op with no response. I think it's good as is then

Reply:
Thanks to both!!